ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/JiraIntegrationForm.tsx

```diff
@@ -85,19 +86,19 @@ export const validationSchema = yup.object().shape({
 const defaultSeverities = [
     {
         severity: 'CRITICAL_SEVERITY',
-        priorityName: 'P0-Highest',
```

Comment:
Change to `priorityName` affects new Jira integrations.
* From frontend viewpoint, I am unable to comment about compatibility.
* Asking, not necessarily recommending, whether there is a relationship between new checkbox and existing `FieldArray` of priority mappings? For example, disable editing?
    Kyle, as a general principle, for combined backend and frontend contributions, corresponding screen grabs before and after code changes is very helpful.
* Integration test is not affected because it does not assert or change priority mappings. If there is conditional rendering related to the new checkbox, what is pro and con to add it to integration test?

Reply:
> screen grabs before and after code changes is very helpful.
I am happy to get these for this PR if they would still be helpful.

Reply:
> relationship between new checkbox and existing FieldArray of priority mappings

Changing the default values of the mapping is not really related to the motivation for the addition of the checkbox, but the FieldArray should be greyed out when the "disable setting priority" box is checked since the mapping is ignored.


> what is pro and con to add it to integration test

Are you referring to the cypress tests?

Reply:
Aha, yes cypress tests.

Reply:
You can skip before pictures. For historical record, the following are interesting:
1. create integration
    * **unchecked** option to see priority mapping **enabled**
    * **checked** option to see priority mapping disabled
2. view integration (can be same integration from step 1)
    * **unchecked** option to see both checkbox and priority mapping disabled
    * **checked** option to see both checkbox and priority mapping disabled

Reply:
ok I finally added the suggested screenshots!